botbuilder-teams-js/package.json

```diff
@@ -12,9 +12,10 @@
   },
   "typings": "./lib/index.d.ts",
   "dependencies": {
-    "adaptivecards": "^1.0.0",
-    "botbuilder": "4.3.2",
-    "botframework-connector": "4.3.2"
+    "adaptivecards": "^1.0.0",    
+    "botbuilder": "^4.3.2",
+    "botframework-connector": "^4.3.2",
```

Comment:
shouldn't change the version of `botbuilder` and `botframework-connector`. The current version should lock down with botbuilder 4.3.2.

Reply:
ah, I did it wrong when merging, will fix

Reply:
done